Lib/distutils/util.py

```diff
@@ -92,25 +92,17 @@ def get_host_platform ():
 
     return "%s-%s-%s" % (osname, release, machine)
 
-# get_host_platform ()
-
-def get_platform ():
+def get_platform():
     TARGET_TO_PLAT = {
         'x86' : 'win32',
         'x64' : 'win-amd64',
         'arm' : 'win-arm32',
     }
 
-    targetPlatformFromEnvironment = os.environ.get('VSCMD_ARG_TGT_ARCH')
-
-    if targetPlatformFromEnvironment != None and targetPlatformFromEnvironment in TARGET_TO_PLAT:
-        targetPlatform = TARGET_TO_PLAT[targetPlatformFromEnvironment]
+    if os.name == 'nt':
```

Comment:
Let's only define `TARGET_TO_PLAT` for Windows as well. Even though it's static, it still gets constructed on each call and it doesn't have any meaning on other OS's.